modules/rubiconBidAdapter.js

```diff
@@ -176,166 +286,57 @@ export const spec = {
   buildRequests: function (bidRequests, bidderRequest) {
     // separate video bids because the requests are structured differently
     let requests = [];
-    const videoRequests = bidRequests.filter(bidRequest => bidType(bidRequest) === 'video').map(bidRequest => {
-      bidRequest.startTime = new Date().getTime();
-
-      const data = {
-        id: bidRequest.transactionId,
-        test: config.getConfig('debug') ? 1 : 0,
-        cur: ['USD'],
-        source: {
-          tid: bidRequest.transactionId
-        },
-        tmax: bidderRequest.timeout,
-        imp: [{
-          exp: config.getConfig('s2sConfig.defaultTtl'),
-          id: bidRequest.adUnitCode,
-          secure: 1,
```

Comment:
This was hard coded before to `1`

Do we want to remove this and never pass it?

Reply:
Found a message in chat from @bretg 

> So looked at `secure` - that should come from someone explicitly setting `ortb2.secure`, not automatic

Reply:
👍